packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -28,6 +32,7 @@ systemd:
             Environment="ETCD_PEER_CLIENT_CERT_AUTH=true"
             Environment="RKT_RUN_ARGS=${etcd_arch_rkt_args}"
             Environment="${etcd_arch_options}"
+            ExecStartPre=/opt/etcd-private-interface
```

Comment:
This is not needed anymore, right?

Reply:
Oh yes, this is not needed. But I was testing @rata 's hypothesis and added all the changes in a separate commit.

I will clean it up if what he is saying works. Otherwise we will be back to what existed before.

Reply:
Oh, more confused. Does it work or not? I see this comment is more recent that the one that says it doesn't :joy: